lib/model/security/token.js

```diff
@@ -34,6 +34,16 @@ class Token {
     this.jwt = data.jwt || null;
     this.refreshed = Boolean(data.refreshed);
   }
+
+  get type() {
+    if (this.jwt && this.jwt.startsWith(Token.APIKEY_PREFIX)) {
```

Comment:
🤔 is this system going to work when upgrading Kuzzle on a system that already contains tokens and API keys?
Maybe not, but in the meanwhile, such a Kuzzle would not tell the difference between API keys and tokens, so it's not bad.

Reply:
Yes this will work after an upgrade even if there are tokens and API Keys contained, because tokens and api keys are already stored with a prefix (`kauth-` for tokens, and `apikey-` for api keys) this means that even if Kuzzle get's an upgrade and is restarted, tokens and apikeys will still be stored with their prefix.